test/integration/app-link-types/tsconfig.json

```diff
@@ -1,26 +0,0 @@
-{
```

Comment:
is this deleted by accident? you added them in https://github.com/vercel/next.js/pull/45879

Reply:
@huozhi there was a renaming for that test but the tsconfig was ignored so the thing kept in my local system. When I removed that from gitignore the thing was committed accidentally. As explained in the description:

> test/integration/app-link-types was a non-existing test that I accidentally committed previously.